src/main/java/frc/robot/systems/AutoPathChooser.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.systems;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.systems.ElevatorArmFSM.FSMState;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+
+public class AutoPathChooser {
+	private static SendableChooser<FSMState> autoPathChooser;
+	private static SendableChooser<Integer> nodeChooser;
+
+	/**
+	 * Constructor for creating the AutoPathChooser class. It is used in order to select the
+	 * auto path and node for the robot without re-deploying.
+	 */
+	public AutoPathChooser() {
+		autoPathChooser = new SendableChooser<>();
+		autoPathChooser.setDefaultOption("Path 1", FSMState.IDLE);
```

Comment:
We're going to rename these paths right?

Reply:
Yes! That code is just template [Resolved]